config/elasticsearch.yml

```diff
@@ -37,3 +37,6 @@ searchguard.ssl.transport.truststore_alias: ca
 # searchguard.ssl.http.keystore_alias: cert
 # searchguard.ssl.http.truststore_filepath: certs/truststore.jks
 # searchguard.ssl.http.truststore_alias: ca
+
+# TODO: Evaluate whether it's worth paying for X-Pack advanced features. Basic monitoring is free.
+xpack.security.enabled: false
```

Comment:
Security module is an advanced feature in x-pack. Security will be disabled in 30 days unless we get the license so I disabled it to start with. If enabled it'd require extra configuration which we aren't sure we gonna be using. The only one which remain free to use is monitoring.